jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MimeTypes.java

```diff
@@ -37,6 +40,40 @@
 public class MimeTypes
 {
     static final  Logger LOG = LoggerFactory.getLogger(MimeTypes.class);
+    private static final Index<Locale> KNOWN_LOCALES = new Index.Builder<Locale>()
+        .caseSensitive(false)
+        .withAll(() -> Arrays.stream(Locale.getAvailableLocales())
+            .collect(Collectors.toMap(Locale::toString, UnaryOperator.identity())))
```

Comment:
Don't use the unreliable `Locale.toString()` here which is for end users, use the `Locale.toLanguageTag()` which uses reliable language tags.

```suggestion
            .collect(Collectors.toMap(Locale::toLanguageTag, UnaryOperator.identity())))
```

Reply:
Strange, I had tried that, but was originally getting long form strings like getDisplayName.   Must have been my-bad somehow.   OK will use this!